source/taup/taup_slowplot.rst

```diff
@@ -0,0 +1,36 @@
+taup slowplot
+=============
+
+:本节贡献者: |田冬冬|\（作者）、
+             |姚家园|\（审稿）
+:最近更新日期: 2021-06-01
+
+----
+
+``taup slowplot`` 可以很方便地生成一个可用于绘制慢度-深度剖面的 GMT 脚本。
+
+此处的慢度是指 :math:`p = \frac{r}{v}`\ ，单位为 s/rad。对于任意射线而言，其
+射线参数（或水平慢度）为 :math:`p = \frac{r \sin\theta}{v}`\ ，其中
+:math:`\theta` 是射线相对于地面垂线的角度。由于 :math:`\sin\theta \leq 1`\，
+因而慢度-深度模型展示了某个射线参数对应的射线所能采样的最大深度。
+
```

Comment:
> Put the paragraph above in a note directive? 

This paragraph describes slowness. I think it should be part of the tutorial, rather than a note.

> Besides, shouldn't it be near the last paragraph. They are related to each other.

When I try the slowplot command, the first question in my mind is, what the definition of slowness in TauP. That's why I put the description at the start of the tutorial.

Reply:
Not really. I think we should focus on the tutorial instead of the seismological basis.